syntax = "proto3";
import "radix_tree.proto";
package proto_msg;



option java_package = "com.github.cristobalm.proto";
option java_outer_classname = "Entities";

message EntitiesMapping{
    RadixTree entities_mapping = 1;

    uint64 subjects_count = 2;
    uint64 predicates_count = 3;
    uint64 objects_count = 4;
}
